ingesters/installation_instructions_template

```diff
@@ -1,20 +1,26 @@
 To install the Debian package, make sure the Gravwell Debian repository is configured [as described in the quickstart](debian_repo). Then run the following command as root:
 
+:::::{div}
 ::: {parsed-literal}
 apt update && apt install {{package}}
 :::
+:::::
 
 To install the Redhat package, make sure the Gravwell Redhat repository is configured [as described in the quickstart](redhat_repo). Then run the following command as root:
 
+:::::{div}
 ::: {parsed-literal}
 yum install {{package}}
 :::
+:::::
 
 To install via the standalone shell installer, download the installer from the [downloads page](/quickstart/downloads), then run the following command as root, replacing X.X.X with the appropriate version:
 
+:::::{div}
 ::: {parsed-literal}
 bash {{standalone}}_installer_X.X.X.sh
 :::
+:::::
```

Comment:
Due to how the copy button is placed, I added a colon-code-fence around the existing colon-code-fence in order to create a `div`